commit/core.pyx

```diff
@@ -676,13 +676,24 @@ cdef class Evaluation :
         print '   [ %s ]' % ( time.strftime("%Hh %Mm %Ss", time.gmtime(self.CONFIG['optimization']['fit_time']) ) )
 
 
-    def save_results( self, path_suffix = None ) :
+    def save_results( self, path_suffix = None, save_pickle = True, save_txt = False ) :
         """Save the output (coefficients, errors, maps etc).
 
         Parameters
         ----------
         path_suffix : string
             Text to be appended to "Results" to create the output path (default : None)
+        save_pickle : boolean
+            Save everything in a pickle file containing the following list L:
+                L[0]: dictionary with all the configuration details
+                L[1]: np.array obtained through the optimisation process with the normalised kernels
+                L[2]: np.array renormalisation of L[1]
+            (default : True)
```

Comment:
I would say to set everything to `False` by default, so that it is consistent with before.

Reply:
With the proposed defaults it is already backward compatible.

Reply:
You're right, I had misunderstood the `save_pickle` field, I though you had added something to it